### PR TITLE
roger - CEM-2650 - first version of flash screen effect

### PR DESCRIPTION
**Pull request by roggc**
https://wolfbeacon.atlassian.net/browse/CEM-2650
flashScreenEffect

### CODE REVIEW

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+export interface IScreenFlashEffectProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    veryBigNumber?: number;
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:7]: Missing typescript documentation

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    opacity?: number;
+    animationDurationInMs?: number;
+    numberOfTimesForAnimation?: number;
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:10]: NumberOfFlashes?

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+const FlashScreen = styled.div<IScreenFlashEffectProps>`
+    ${({
+        veryBigNumber,
+        opacity,
+        animationDurationInMs,
+        numberOfTimesForAnimation,
+        color,
+    }): string => `
+    @keyframes flash {
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:44]: Fix indentation here

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Comments needed

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx (hunk 3 above)
Comment (roggc) [reply src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:48]: I don't understand, what's wrong with indentation. prettier formatted gives me that.

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+export interface IScreenFlashEffectProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    /** a big number for the width, height and z-index of the flash screen */
+    veryBigNumber?: number;
+    opacity?: number;
+    animationDurationInMs?: number;
+    numberOfFlashes?: number;
+    /** the hexadecimal code for color of flash screen, default: ff0000 */
+    color?: string;
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:13]: flashColor

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 5]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+export interface IScreenFlashEffectProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    /** a big number for the width, height and z-index of the flash screen */
+    veryBigNumber?: number;
+    opacity?: number;
+    animationDurationInMs?: number;
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:10]: flashDuration 

Put MS in Typescript documentation

File: src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import styled from 'styled-components';
+
+export interface IScreenFlashEffectProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    /** a big number for the width, height and z-index of the flash screen */
+    veryBigNumber?: number;
+    opacity?: number;
```
Comment (ralph-dev) [inline src/Containers/ScreenFlashEffect/ScreenFlashEffect.tsx:9]: flashOpacity

**Review by ralph-dev — APPROVED** [review]
Lgtm needs more docs and better naming

**Comment by roggc** [comment]
@ralph-dev, may I merge despite the failing tests?

**Pull request merged**